### PR TITLE
Add some terms related to styles to the glossary

### PR DESCRIPTION
**Pull request by oandregal**
Adds some terms related to styles to the glossary with links to existing documentation: block styles, block supports, global styles.

### CODE REVIEW

File: docs/explanations/glossary.md [hunk 1]
```diff
@@ -52,6 +64,14 @@ A type of block where the content of which may change and cannot be determined a
 
 This refers to a collection of features that ultimately allows users to edit their entire website using blocks as the starting point. This feature set includes everything from block patterns to global styles to templates to design tools for blocks (and more). First released in WordPress 5.9.
 
+## Global Styles
+
+The CSS styles generated by WordPress and enqueued as a embedded stylesheet in the front end of the site. The stylesheet ID is `global-styles-inline-css`. The contents of this stylesheet come from the default `theme.json` of WordPresss, the `theme.json` theme, and the styles provided by the user via the global styles sidebar in the site editor.
```
Comment (ntsekouras) [inline docs/explanations/glossary.md]:
```suggestion
The CSS styles generated by WordPress and enqueued as an embedded stylesheet in the front end of the site. The stylesheet ID is `global-styles-inline-css`. The contents of this stylesheet come from the default `theme.json` of WordPress, the theme's `theme.json`, and the styles provided by the user via the global styles sidebar in the site editor.
```

**Review by ntsekouras — APPROVED** [review]
Thanks @oandregal 💯

File: docs/explanations/glossary.md [hunk 2]
```diff
@@ -12,6 +12,18 @@ The object representation of the current state of a block in post content. When
...
 
+## Block Styles
+
+The CSS styles that are part of the block, either via its stylesheet or via the block markup itself. For example, a class attached to the block markup is considered block styles.
+
+Compare to <a href="#global-styles">Global Styles</a>. In contraposition to "Global Styles", block styles are sometimes referred to as "Local Styles".
+
+## Block Supports
+
+An API for blocks to declare what feautures they support. By declaring support for a feature, the API would add aditional <a href="#attributes">attributes</a> to the block and matching UI controls for most of the existing block supports.
```
Comment (getdave) [inline docs/explanations/glossary.md]:
```suggestion
An API for blocks to declare what features they support. By declaring support for a feature, the API would add additional <a href="#attributes">attributes</a> to the block and matching UI controls for most of the existing block supports.
```

File: docs/explanations/glossary.md (hunk 1 above)
Comment (getdave) [inline docs/explanations/glossary.md]:
```suggestion
The CSS styles generated by WordPress and enqueued as a embedded stylesheet in the front end of the site. The stylesheet ID is `global-styles-inline-css`. The contents of this stylesheet come from the default `theme.json` of WordPress, the `theme.json` theme, and the styles provided by the user via the global styles sidebar in the site editor.
```

**Review by getdave — APPROVED** [review]
Spelling nits.

File: docs/explanations/glossary.md [hunk 3]
```diff
@@ -12,6 +12,18 @@ The object representation of the current state of a block in post content. When
 
 The abstract term used to describe units of markup that, composed together, form the content or layout of a webpage. The idea combines concepts of what in WordPress today we achieve with shortcodes, custom HTML, and embed discovery into a single consistent API and user experience.
 
+## Block Styles
+
+The CSS styles that are part of the block, either via its stylesheet or via the block markup itself. For example, a class attached to the block markup is considered block styles.
+
+Compare to <a href="#global-styles">Global Styles</a>. In contraposition to "Global Styles", block styles are sometimes referred to as "Local Styles".
```
Comment (mcsf) [inline docs/explanations/glossary.md:19]: > Local Styles

Would it be useful to add an entry in this very glossary like so:

```markdown
## Local Styles

See <a href="#block-styles">Block Styles</a>.
```

Comment (oandregal) [reply docs/explanations/glossary.md:19]: Oh, missed this comment, sorry. https://github.com/WordPress/gutenberg/pull/38083

**Pull request merged**